rasa/core/channels/socketio.py

```diff
@@ -177,8 +191,20 @@ async def health(_: Request) -> HTTPResponse:
             return response.json({"status": "ok"})
 
         @sio.on("connect", namespace=self.namespace)
-        async def connect(sid: Text, _: Any) -> None:
-            logger.debug(f"User {sid} connected to socketIO endpoint.")
+        async def connect(sid: Text, environ: Dict, auth: Optional[Dict]) -> None:
+            if self.jwt_key:
+                jwt_payload = None
+                if auth and auth.get("token"):
+                    jwt_payload = _decode_bearer_token(
+                        auth.get("token"), self.jwt_key, self.jwt_algorithm,
+                    )
+
+                if jwt_payload:
+                    logger.debug(f"User {sid} connected to socketIO endpoint.")
+                else:
+                    return False
```

Comment:
return `False` and the type annotation says `-> None`?

Reply:
Ah sorry, being sloppy